### PR TITLE
Ashirwad patch 1

### PR DESCRIPTION
**Pull request by nathancday**

### CODE REVIEW

**Comment by nathancday** [comment]
Hey @ijlyttle this is a docs change, do you think the verbage is good enough to squ-erge?

File: R/boxr_search.R [hunk 1]
```diff
@@ -10,20 +10,21 @@
...
+#'  * Reserved words for boolean operations
+#'    - `AND`, `OR`, and `NOT` (uppercase only) are interpreted as special context e.g.
+#'    `box_search("NOT this")`, `box_search("this AND that")`
 #'       
-#'  * Phrases can be searched
-#'    - by surrounding them with "quotation marks" in the query string e.g. 
-#'      `box_search("match similar phrases")`, `box_search("\"match this phrase exactly\"")`
+#'  * Exact phrases can be matched
+#'    - by surrounding them with double quotation marks e.g.
+#'    `box_search('"this exact phrase"')` or `box_search("\"this exact phrase\"")`
```
Comment (ashirwad) [inline R/boxr_search.R:24]: @nathancday, `box_search('"this exact phrase"')` does not perform an exact search. I tested it on my computer, but you can verify at your end. So, adding this example will be misleading.

Comment (nathancday) [reply R/boxr_search.R:24]: Looks like we are getting different results. 🤷

Comment (ashirwad) [reply R/boxr_search.R:24]: Huh, that's interesting! Then leave the example there. :)

**Pull request merged**